tests/integration/ovs_test.py

```diff
@@ -17,20 +17,32 @@
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
 
+from contextlib import contextmanager
+
 import pytest
 
+import libnmstate
 from libnmstate.schema import Interface
 from libnmstate.schema import InterfaceIPv4
+from libnmstate.schema import InterfaceState
+from libnmstate.schema import InterfaceType
 from libnmstate.schema import OVSBridge
 from libnmstate.error import NmstateLibnmError
 
 from .testlib import assertlib
+from .testlib import cmd as libcmd
 from .testlib.ovslib import Bridge
 
 
 BRIDGE1 = "br1"
 PORT1 = "ovs1"
 
+DNF_REMOVE_NM_OVS_CMD = ("dnf", "remove", "-y", "-q", "NetworkManager-ovs")
+
+DNF_INSTALL_NM_OVS_CMD = ("dnf", "install", "-y", "-q", "NetworkManager-ovs")
+
+SYSTEMCTL_RESTART_NM_CMD = ("systemctl", "restart", "NetworkManager")
```

Comment:
nit: The two newlines seem to be redundant.

Reply:
Done! Thanks!